### DISCUSSION TITLE
Responsive Column width

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#1241] [Q&A (Help)]
<img width="773" alt="Screenshot 2024-11-24 alle 16 14 36" src="https://github.com/user-attachments/assets/89a3d26e-7aab-4739-86d1-ee709739b24c">

I'm not sure that backup is optimizing columns..
Some feedback ? :)

[a comment by an automated account is not shown]

**Comment by karandatwani92** [comment]
Hey @blondie63

Datatable or CSS does not trim the text length here. It is being shortened on the server side. You can change the length via:
```php
CRUD::column('body')->limit($characters = 100);
```

-- AND-- 

If you want to modify the `<table class="">` tag classes, that is doable by theme config:
**Step 1)** publish theme config
```
php artisan vendor:publish --tag="theme-tabler-config"
```
**Step 2)** Replace table classes via config.
```php
'table' => "table table-striped table-hover rounded card-table table-vcenter card-table shadow-xs border-xs display compact",
```

![Screenshot 2024-11-25 at 3 01 33 PM](https://github.com/user-attachments/assets/e483ccb9-4def-481e-a99b-97ee34386b0b)
